src/chart/MultiChart.js

```diff
@@ -115,6 +115,19 @@
         this._chartTypeProperties = _;
         return this;
     };
+    
+    MultiChart.prototype.getChartDataType = function () {
+        var context = this;
+        var types = ["1D","2D","ND"];
+        for(var i=0;i<types.length;i++) {
+            var idx = "_"+types[i]+"ChartTypes";
+            for(var j=0;j<context[idx].length;j++) {
+                if(context[idx][j].id === context.chartType()){
+                    return types[i];
+                }
+            }
+        }
+    };
```

Comment:
or...
Change the prototypes to include the "family" (either by hand) or by:
```
MultiChart.prototype._1DChartTypes = [
        { id: "SUMMARY", display: "Summary", widgetClass: "chart_Summary" },
        { id: "C3_GAUGE", display: "Gauge (C3)", widgetClass: "c3chart_Gauge" }
    ].map(function(item) { item.family = "1D"; return item;});
```
Type of thing?